test/video_helper_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
```

Comment:
This is great, thank you! :+1: 

Can you please also add an integration test with Lotus::View? See https://github.com/lotus/helpers/tree/master/test/integration

It's important, because these helpers will be used in combination with views that perform automatic escape of contents. We must render a real template and assert that the generated HTML is correct. :wink: